packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/cancel-resiliation/translations/Messages_es_ES.json

```diff
@@ -1,3 +0,0 @@
-{
-  "autorenew_service_cancel_resiliation_success": "Se ha cancelado la baja del servicio: {{ service }}."
```

Comment:
Moved to `packages/manager/modules/billing/src/components/cancel-resiliation/translations/Messages_es_ES.jsonù